[rocprof-compute] Run roofline test on GPU 0 by default (#1390)
ROCm/rocm-systems: projects/rocprofiler-compute/CHANGELOG.md

* Default rocprof interface changed from rocprofv3 to rocprofiler-sdk
  * Use ROCPROF=rocprofv3 to use rocprofv3 interface

> * Changed default behavior to run roofline on GPU 0 instead of all GPUs

Reviewer: ```suggestion
* Roofline analysis now runs on GPU 0 by default instead of all GPUs.
```
